readme.md

```diff
@@ -80,6 +81,14 @@ referred to as `Playing the Hitler card`.
 
 ***
 
+### [Parkinson's law](https://en.wikipedia.org/wiki/Parkinson%27s_law)
+
+> Parkinson's law is the adage that "work expands so as to fill the time available for its completion". It is also sometimes applied to the growth of the bureaucratic apparatus in an organization.
```

Comment:
This should be shorter. Preferably only one small sentence that captures the essence of the entry